packages/next/build/webpack-config.ts

```diff
@@ -720,7 +724,7 @@ export default async function getBaseWebpackConfig(
       'use-subscription',
       require.resolve('next', { paths: [dir] })
     ),
```

Comment:
I think it would make sense to not hardcode react dependencies like `scheduler` and `object-assign`, but instead only call `getPackagePath` for `react` and follow all dependencies listed in the `package.json` of e. g. react and add them.